javascript/ql/src/semmle/javascript/XML.qll

```diff
@@ -34,6 +34,8 @@ abstract class XMLLocatable extends @xmllocatable {
  * both of which can contain other elements.
  */
 class XMLParent extends @xmlparent {
+  XMLParent() { this instanceof @xmlelement or xmlEncoding(this, _) }
```

Comment:
Should we add a comment here or in the extractor about why we do it like this?

Reply:
Yeah, probably a good idea. Done.